docs/ref/release_notes.rst

```diff
@@ -0,0 +1,54 @@
+What's new in Iris-grib v0.13
```

Comment:
We should have a consistent way of rendering `iris-grib` everywhere. My preference is for all lower-case and hyphen-separated (i.e. "iris-grib"), as that's how it's referred to elsewhere, notably in the README.